Library/Formula/scirius.rb

```diff
@@ -0,0 +1,49 @@
+class Scirius < Formula
+  desc "Rules manager for Suricata IDPS"
+  homepage "https://github.com/StamusNetworks/scirius"
+  url "https://github.com/StamusNetworks/scirius"
```

Comment:
Please link to a specific release, e.g. `https://github.com/StamusNetworks/scirius/archive/scirius-1.0.tar.gz`